src/main/java/com/github/maven_nar/NarUtil.java

```diff
@@ -386,7 +386,7 @@ public static void makeExecutable(final File file, final Log log) throws MojoExe
     if (file.isFile() && file.canRead() && file.canWrite() && !file.isHidden()) {
       // chmod +x file
       final int result = runCommand("chmod", new String[] {
-          "+x", file.getPath()
+          "+x", file.getPath().replaceAll(" ", "\\ ")
```

Comment:
Because `chmod` is called directly rather than using shell `sh` additional escaping or quoting for spaces shouldn't be required.
Was this required or "added for safety" ?